ember-toucan-core/src/components/form/textarea-field.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col">
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-textarea-label>{{@label}}</field.Label>
+
+    {{#if @hint}}
+      <field.Hint id={{field.hintId}} data-textarea-hint>{{@hint}}</field.Hint>
+    {{/if}}
+
+    <field.Control>
+      <textarea
+        aria-invalid={{if @error "true"}}
+        aria-describedby="{{if @error field.errorId}} {{if @hint field.hintId}}"
+        class="block h-20 min-h-6 p-1 bg-overlay-1 focus:outline-none rounded-sm shadow-focusable-outline focus:shadow-focus-outline
+          {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
+        disabled={{@isDisabled}}
```

Comment:
For sure - those should totally work! I think for this case it _somewhat_ depends.  We are testing out some designs where the label and hint text may also look different based on the disabled state.  If we do in fact change the style of these surrounding elements as well, I had thought that maybe a component argument would be easiest to support as then we can check `@isDisabled` rather than having to reach into the `<textarea>` and pull out the disabled attribute; however, if we are **not** going that route and _only_ styling the disabled `<textarea>`, then I 100% agree with using the `disabled:` selector for sure.

So for example:

```hbs
<field.Label class="{{if @isDisabled 'some-custom-styling'}}" for={{field.id}} data-textarea-label>{{@label}}</field.Label>
```

So I guess, at this point in time, we aren't quite sure exactly where we will land from a styling point here, so I think we have some flexibility for the time being.  We have another task to circle back through all of the components and update them to match designs post-usability testing though.  We are probably 2-3 weeks away from this happening though.  What do you feel is best here knowing above?  I'm still on the fence!  Could go either way.

Reply:
> If we do in fact change the style of these surrounding elements as well, I had thought that maybe a component argument would be easiest to support

Yeah, makes sense in this case!

> having to reach into the <textarea> and pull out the disabled attribute

No, we should _definitely_ not do that! This would likely not be reactive (you won't get rerenders in parts that depend on this, when `disabled` changed). Also it's basically the inverse of the paradigm of all modern JS Apps, with the DOM being a function of state, and not vice versa. 

> What do you feel is best here knowing above? I'm still on the fence! Could go either way.

Let's keep it as it is then! We can still refactor when it becomes clear we won't need it.